### DISCUSSION TITLE
Add a button for Scheduled/MR scripts to save and execute

### DISCUSSION THREAD

**Discussion by bodawui** [burkybang/SuiteAdvanced-Feedback#110] [Ideas]
I have noticed that M/R scripts are now having 2 buttons Save and save and edit.

But I believe that it is benefitial to have there more then save and edit button save and execute.

for developers a loot of free hands

**Comment by burkybang** [comment]
I've attempted this before, but I think I'll try again. It would be extremely useful.

**Comment by burkybang** [comment]
This'll be a feature in the next release. Thanks for the motivation to get this one done.

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/dca30cc0-6207-4683-b947-b22c7a0583dd)

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/942b56ee-e6c9-4f7d-906f-5e41a14c93a7)

**Reply by burkybang** [reply]
Release notes: https://suiteadvanced.com/version-29_0_0
